api_tests/nodes/views/test_node_forks_list.py

```diff
@@ -10,8 +11,11 @@
     ForkFactory
 )
 from rest_framework import exceptions
+from website import mails
 from website.util import permissions
 
+from api.nodes.serializers import NodeForksSerializer
+mails.send_mail = mock.MagicMock()
```

Comment:
Talked in person, summary of comment:
1. Mostly concerned about it being top level-- every other MagicMock is nested within a method in a class AFAICT.
2. The other cases of mocking send_mail do use the decorator, should work but if it isn't working I can take a look.